ovve_ui/ovve_ui.py

```diff
@@ -212,19 +212,21 @@ def display(self, i) -> None:
 
     def update_ui_params(self, params: Params) -> None:
         self.params = params
-        self.logger.info(self.params.to_JSON())
-        self.updateMainDisplays()
-        self.updateGraphs()
+        if self.params.run_state > 0:
+            self.logger.info(self.params.to_JSON())
+            self.updateMainDisplays()
+            self.updateGraphs()
 
     def update_ui_alarms(self, alarms_dict: dict) -> None:
-        self.alarms = Alarms()
-        self.alarms.from_dict(alarms_dict)
-        self.logger.info(self.alarms.to_JSON())
-
-        for i in range(len(alarms_dict)):
-            if list(alarms_dict.items()
-                    )[i][1]:  #TODO: Revisit this for multi alarm handling
-                self.showAlarm(i)
+        if self.params.run_state > 0:
```

Comment:
I don't think this is right.  We may have alarms that fire even if not running.  Ex. AC disconnected, low battery, etc

Reply:
Ah yes, you're right. Perhaps we will need a way to check which alarms are 'reasonable' to trigger when stopped. I fixed this below.